File: scripts/fibonacci.py
import sys

def fibonacci(number: int):
    if number < 2:
        return number

    return fibonacci(number - 1) + fibonacci(number - 2)

if __name__ == '__main__':
    if len(sys.argv) < 2:
        print("Provide an argument for fibonacci sequence", file=sys.stderr)
        sys.exit(1)

    n = int(sys.argv[1])
    for i in range(1, n + 1):
        print(f"Fibonacci sequence number at index {i} is {fibonacci(i)}")

File: scripts/php/fibonacci.php
<?php

function fibonacci($number) {
    if ($number < 2) {
        return $number;
    }

    return fibonacci($number - 1) + fibonacci($number - 2);
}

if (count($argv) < 2) {
    fwrite(STDERR, "Provide an argument for fibonacci sequence" . PHP_EOL);
    exit(1);
}


$n = intval($argv[1]);
foreach (range(1, $n) as $i) {
    $result = fibonacci($i);
    echo "Fibonacci sequence number at index $i is $result" . PHP_EOL;
}
